app/code/Magento/Backend/view/adminhtml/templates/dashboard/totalbar/refreshstatistics.phtml

```diff
@@ -8,21 +8,13 @@
     <div class="page-actions-inner">
         <div class="page-actions-buttons">
             <?= $block->getChildHtml() ?>
-
-            <form class="action-element"
```

Comment:
I would recommend another solution here. Removing the form and form key looks not very good considering the fact that the current approach works well on 2.2-develop

Reply:
I've reverted and adjusted the implementation of RefreshStatistics controller.
Thank you.